app/src/main/java/com/keylesspalace/tusky/components/announcements/AnnouncementAdapter.kt

```diff
@@ -60,6 +63,11 @@ class AnnouncementAdapter(
     override fun onBindViewHolder(holder: BindingHolder<ItemAnnouncementBinding>, position: Int) {
         val item = items[position]
 
+        val local = Locale.getDefault()
+        val format = DateFormat.getBestDateTimePattern(local, "yMMMdHmm")
+        holder.binding.announcementDate.text = SimpleDateFormat(format, local).format(item.publishedAt)
```

Comment:
Why not use `AbsoluteTimeFormatter` for consistency with other parts of the app?

Reply:
Oh, yes, indeed. I will fix the code!